cloudinit/config/schemas/schema-cloud-config-v1.json

```
@@ -1480,6 +1480,11 @@
           ],
           "additionalProperties": false,
           "properties": {
+            "install_source": {
+              "type": "string",
+              "pattern": "^(distro|ppa:.+)$",
+              "description": "Whether to install landscape-client deb package from the 'distro' -updates pocket or a custom 'ppa:YOUR_PPA'. Default: 'distro'"
```

Nit: As this used gets rendered in cloud-init docs, we could probably be less specific for the sake of the less ubuntu-versed users.

```suggestion
              "description": "Whether to install landscape-client deb package from the 'distro' repos or a custom 'ppa:YOUR_PPA'. Default: 'distro'"
```